.github/workflows/ci.yml

```diff
@@ -16,6 +16,13 @@ env:
   MAVEN_TEST: "-B -Dair.check.skip-all --fail-at-end"
   RETRY: .github/bin/retry
 
+# Cancel all workflow runs except latest within each concurrency group.
+concurrency:
```

Comment:
I wasn't aware of that feature. Neat!

Reply:
It's in beta (but stable enough for our use-case).

Reply:
Yeah - this is nice feature.